src/js/modules/sections/components/SectionPage.js

```diff
@@ -65,10 +77,13 @@ const SectionPage = ({ classes,
     <div className={classes.SectionPage}>
       <h1 className={classes.sectionName}>{section.name}</h1>
       {
-        directSubsections !== null &&
-        <ul className={classes.subsectionBar}>
-          {createLinksToDirectSubsections()}
-        </ul>
+        isObjectEmpty(directSubsections) ? (
```

Comment:
Wait, if the direct subsections *are* empty you create links to them? Also update function to correct format for creating links